refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,758 @@
+| `date` | 1 | Date and Time when Consent was given in JSON Schema [date-time](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `expires` | 0-1 | Date and Time when Consent expires in JSON Schema [date-time](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `target` | 0-1 | Optionally one of {`ORGANISATION`, `PARTNERS`, `SYSTEM`} to indicate the category of Systems to which consent for processing is given. In absence of indication `SYSTEM` is assumed. |
+| `scope` |  0-1 | a [Privacy Scope](#privacy-scope) in absence of which the Consent SHOULD be interpreted as unlimited |
+| `replaces` |  0-* | Optionally one or more 'consent-id's of previous consents that have became void when this consent was made |
+| `replaced-by` |  0-* | Optionally one or more 'consent-id's of previous consents that have became void when this consent was made |
+
+### Data Capture
+
+A Data Capture is given by one Data Subject which can be identified by one or more [Data Subject Identities](#decentralized-identity-of-data-subjects).
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `data-subject` |  1-* | [Data Subject Identities](#decentralized-identity-of-data-subjects) each containing one `dsid` and one `dsid-schema`|
+| `capture-id` | 1 | a string in the [uuid](https://www.rfc-editor.org/rfc/rfc4122.html) format |
+| `target` | 0-1 | Optionally one of {`ORGANISATION`, `PARTNERS`, `SYSTEM`}. In absence of indication `SYSTEM` is assumed |
+| `fragments` | 1-* | One or more [Data Capture Fragments](#data-capture-fragments) |
+
+A Data Capture concerns one and only one Data Subject who MAY be identified by multiple Data Subject Identities.
+
+#### Data Capture Fragments
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `fragment-id` | 1 | a string in the [uuid](https://www.rfc-editor.org/rfc/rfc4122.html) format |
+| `selector` | 1 | a string used to uniquely identify a data field (in the System's data model) to which the fragment corresponds |
+| `date` | 1 | Date and Time when data was Captured was given in JSON Schema [date-time](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `scope` |  0-1 | a [Privacy Scope](#privacy-scope) in absence of which the fragment SHOULD be interpreted as unlimited |
+| `target` | 0-1 | Optionally one of {`ORGANISATION`, `PARTNERS`, `SYSTEM`}. In absence of indication `SYSTEM` is assumed |
+| `retention` | 1-* | one or more [Retention Policies](#retention-policy) |
+| `provenance` | 1-* | one or more [Provenance](#provenance) |
+| `data` | 0-* | Optionally concrete data (Format **TBD**) |
+| `legal-base` | 0-* | Optionally an array of values among `CONTRACT`, `CONSENT`, `LEGITIMATE-INTEREST`, `NECESSARY`, `OTHER-LEGAL-BASE` |
+
```

Comment:
Consent can't be linked to a particular fragment. 
A consent is given for a particular [Privacy Scope](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#privacy-scope). See `scope` of [Consent](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#consent).
A [Fragment](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#data-capture-fragments) has a `selector`, which is, or is not, a part of the scope of a particular consent (or made legal by some other legal base - such as mandatory keeping regardless fo consent).

A `selector` is a subcategory (of a data Category) which is only one dimention of the Privacy Scope. You can't constraint the Capture (and its Fragment) by a consent, because in fact you may be eligible to collect data, but not to share, or use it, or sell it. Also consents live a life of their own, idenpendent from data capture fragments. You may, for instance : 
- have legal right to collect the data without consent (due to a pre-contractual relationship with the Data Subject)
- the data subject does not go throught the trasnaction with you, and then
- you can still keep the data without consent (under legitiamte interest) hoping they would come back
- They can then Object to their data being used. You can keep the data, you don't have to delete it.
- Finally they can make a DELETE request.

See [this example here](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/expected-behavior.md#updating-the-eligible-privacy-scope-upon-object-and-restrict-privacy-request-demands).

Reply:
Consent can't be linked to a particular fragment. 
A consent is given for a particular [Privacy Scope](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#privacy-scope). See `scope` of [Consent](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#consent).
A [Fragment](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#data-capture-fragments) has a `selector`, which is, or is not, a part of the scope of a particular consent (or made legal by some other legal base - such as mandatory keeping regardless fo consent).

A `selector` is a subcategory (of a data Category) which is only one dimention of the Privacy Scope. You can't constraint the Capture (and its Fragment) by a consent, because in fact you may be eligible to collect data, but not to share, or use it, or sell it. Also consents live a life of their own, idenpendent from data capture fragments. You may, for instance : 
- have legal right to collect the data without consent (due to a pre-contractual relationship with the Data Subject)
- the data subject does not go throught the trasnaction with you, and then
- you can still keep the data without consent (under legitiamte interest) hoping they would come back
- They can then Object to their data being used. You can keep the data, you don't have to delete it.
- Finally they can make a DELETE request.

See [this example here](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/expected-behavior.md#updating-the-eligible-privacy-scope-upon-object-and-restrict-privacy-request-demands).